 --- officecfg/registry/data/org/openoffice/Office/Common.xcu
+++ officecfg/registry/data/org/openoffice/Office/Common.xcu
@@ -523,6 +523,21 @@
           <value >private:image/3216</value>
          </prop>
        </node>
+      <node oor:name="m15" oor:op="replace" install:module="dicooo">
+        <prop oor:name="URL"  oor:type="xs:string">
+          <value >macro:///Launcher.DicOOo.StartDicOOo</value>
+        </prop>
+        <prop oor:name="Title">
+          <value xml:lang="de">Weitere Wörterbücher installieren...</value>
+          <value xml:lang="en-US">Install new dictionaries...</value>
+        </prop>
+        <prop oor:name="TargetName"  oor:type="xs:string">
+          <value >_self</value>
+        </prop>
+        <prop oor:name="ImageIdentifier"  oor:type="xs:string">
+          <value >private:image/3216</value>
+        </prop>
+      </node>
      </node>
    </node>
    <node oor:name="Forms">
